File: daySix/Java/ReplaceAllNegative.java
import java.util.Arrays;

public class ReplaceAllNegative{

    public static int[] replaceNegativeWithZeros(int[] array){
    
        int[] newArray = new int [array.length];
        
        for(int count = 0; count < array.length; count++){
        
            if(array[count] < 0){
            
                newArray[count] = 0;
            
            }else {
            
                newArray[count] = array[count];
            }
        
        }
    
        return newArray;
    }
    
    
    public static void main(String[] args){
    
        int[] array = {5, -9, 3, -6, 2, -11};
    
        System.out.println(Arrays.toString(replaceNegativeWithZeros(array)));
    
    
    }
    
    
} 

   

File: daySix/Python/replace_all_negative.py
def replace_negative_with_zeros(array):

    new_array = [0] * len(array)

    for count in range(len(array)):
    
        if array[count] < 0:
        
            new_array[count] = 0
            
        else:
        
            new_array[count] = array[count]
            

    return new_array



    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
